data/examples/declaration/value/function/if-with-comment-next-to-keyword-out.hs

```diff
@@ -0,0 +1,10 @@
+foo =
+  if x
+    then -- comment 1
+    -- comment 2
```

Comment:
This is the case that you wanted to add, I think. Note that this line is not indented. Not sure if this is what you intended or not. If this is difficult to fix I'd say it is probably fine.